packages/fast-element/docs/building-components.md

```diff
@@ -261,37 +261,45 @@ You can also use an expression to set a property on an HTML element. Simply plac
 </span>
 ```
 
-**Example: Inner HTML**
+**Example: ARIA Attributes**
 
 ```HTML
-<div innerhtml=${x => sanitize(x.someDangerousHTMLContent)}></div>
+<div role="progressbar"
+     aria-valuenow="${x => x.value}"
+     aria-valuemin="${x => x.min}"
+     aria-valuemax="${x => x.max}">
+</div>
 ```
 
-> **WARNING:** Avoid scenarios that require you to directly set HTML, especially when the content is coming from an external source. If you must do this, always sanitize the HTML content using a robust HTML sanitizer library, represented by the use of the `sanitize` function above.
+**Example: Boolean Attributes**
 
-#### Dynamic Attributes
+```HTML
+<button type="submit" ?disabled="${x => !x.enabled}">Submit</button>
+```
 
-Most HTML attributes have a corresponding property on the HTML element itself, so the method of property binding described above will be the most common mechanism used. However, there are some scenarios, such as with [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA), where there is no corresponding element property. For these scenarios, prepend a `$` to the attribute name, and the value will be set with the `setAttribute` API instead of through a property. Additionally, some attributes are known as [boolean attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#Boolean_Attributes)  (e.g. required, readonly, disabled). These attributes behave differently from normal attributes, and need special value handling. The templating engine will handle this for you if you prepend the attribute name with a `?`.
+#### Dynamic Properties
 
-**Example: ARIA Attributes**
+Properties can also be set directly on an HTML element. To do so, prepend the property name with `:` to indicate a property binding. The template engine will then use your expression to assign the element's property value.
+
+**Example: Basic Property Values**
 
 ```HTML
-<div role="progressbar"
-     $aria-valuenow="${x => x.value}"
-     $aria-valuemin="${x => x.min}"
-     $aria-valuemax="${x => x.max}">
-</div>
+<my-element :myCustomProperty=${x => x.mySpecialData}>
+  ...
+</my-element>
 ```
 
-**Example: Boolean Attributes**
+**Example: Inner HTML**
 
 ```HTML
-<button type="submit" ?disabled="${x => !x.enabled}">Submit</button>
+<div :innerHTML=${x => sanitize(x.someDangerousHTMLContent)}></div>
 ```
 
+> **WARNING:** Avoid scenarios that require you to directly set HTML, especially when the content is coming from an external source. If you must do this, always sanitize the HTML content using a robust HTML sanitizer library, represented by the use of the `sanitize` function above.
```

Comment:
FYI I'm looking into taking advantage of new Trusted Types APIs so that anytime someone tries to bind the innerHTML property, we force the update to go through our HTML policy (or one they provide). Our policy would throw by default, forcing end users to provide a sanitization policy.